packages/bpk-component-popover/readme.md

```diff
@@ -101,3 +109,10 @@ const onClose = (event, {
   ...
 }
 ```
+
+## Theme Props
+
+Both all the props from [link](/components/web/typography#links) and
+
+* `modalCloseButtonColor`
```

Comment:
Similarly I have fixed the names of these in my PR (should be `closeButtonColor`, `closeButtonHoverColor` and
`closeButtonActiveColor`)

Reply:
I guess your PR should go in first and I'll update mine later on

Reply:
Yeah the PR is #347